src/Data/LLVM/BitCode/GetBits.hs

```diff
@@ -159,144 +159,151 @@ extractFromByteString :: Int# {-^ the last bit accessible in the ByteString -}
                       -> Int# {-^ the bit to start extraction at -}
                       -> Int# {-^ the number of bits to extract -}
                       -> ByteString {-^ the ByteString to extract from -}
-                      -> Either String (() -> (# Int#, Int# #))
+                      -> Either String (() -> (# Word#, Int# #))
 extractFromByteString !bitLim# !sBit# !nbits# bs =
```

Comment:
Now that we have a `quick` flag, is it worth making `extractFromByteString'` above the `quick` version of this function?